src/types.ts

```diff
@@ -736,6 +746,21 @@ export class ZodNumber extends ZodType<number, ZodNumberDef> {
       ],
     });
 
+  multipleOf = (value: number, message?: errorUtil.ErrMessage) =>
+    new ZodNumber({
+      ...this._def,
+      checks: [
+        ...this._def.checks,
+        {
+          kind: "multipleOf",
+          value: value,
+          message: errorUtil.toString(message),
+        },
+      ],
+    });
+
+  mod = this.multipleOf;
```

Comment:
Not a blocker:

I'm not sure this alias is helpful or clear. It reads a bit like it is transforming the value rather than validating it.

Reply:
Hmm yep, that's fair. Perhaps `.step(x)` to coincide with dom input api?

Reply:
Sure! I personally thing `multipleOf` is perfectly clear and that aliases are noisy/distracting, but that's 100% my personal opinion so if it seems more user-friendly to have something like `step`, that seems better than `mod` to me.

Reply:
Ok, changed name of alias to step for now since I personally like having shorthands around, but really wouldn't mind if you decided to remove it before release.